PIPELINE-README.md

```diff
@@ -0,0 +1,72 @@
+pipeline/pipeline.yml is a CloudFormation template that will deploy all the required pipeline components. Once the stack has deployed the Pipeline will automatically execute and deploy the Serverless Application. See getting started for information on how to deploy the template.
+
+#### Deployed Resources
+
+* Pipeline S3 bucket, used to store pipeline artefacts that are passed between stages.
+* CodePipeline
+* CodeBuild Project
+* Roles for CodePipeline, CodeBuild and the CloudFormation Deployment
+
+### Source
+
+For this application we are hosting our source code in GitHub. Other [Source Integrations](https://docs.aws.amazon.com/codepipeline/latest/userguide/integrations-action-type.html#integrations-source) are available however this template focuses on GitHub. Whenever an update is pushed to the GitHub branch being
+monitored our pipeline will begin executing. The source stage will connect to GitHub using the credentials provided and clone the branch into our pipeline artefact bucket for use in the other stages. 
+
+### Build
+
+In order to run our SAM build and SAM package commands we are using [CodeBuild](https://aws.amazon.com/codebuild/), a fully managed continuous integration service . Codebuild allows us to perform a sequence of commands that we define in the [BuildSpec.yml](https://docs.aws.amazon.com/codebuild/latest/userguide/build-spec-ref.html)
+file that will execute inside the [build environment](https://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref.html) we define using a docker container. For this project we are using the Amazon Linux 2 version 1.0 container with Python 3.7.
+
+Within the buildspec.yml we are:
+
+* Updating SAM to the latest version
+* Running SAM build as per the manual deployment
+* Running SAM Package again as per the manual deployment steps
+* Instructing CodeBuild to pass the output template back to the Pipeline for use in the deployment stage. 
+
+### Deploy
+
+To deploy our application stack we are not using SAM Deploy, instead we are opting to use the CodePipeline native support for CloudFormation. The pipeline has a role it use with appropriate permissions to deploy the template created by the SAM package step which will create a stack containing the resources defined in our SAM Template. We are using [change sets](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-changesets.html) and [approval actions](https://docs.aws.amazon.com/codepipeline/latest/userguide/approvals-action-add.html) to demonstrate a manual approval workflow. The first deployment will not require approval however subsequent updates will.
+
+Additional resources will be deployed as per the main architecture documentation.
+
+## Getting started
+
+To get started you just need to deploy the pipeline CloudFormation stack using the template found in this repository under pipeline/pipeline.yaml in order for this to be successful you will need to provide some additional information.
```

Comment:
Adding the commands to package and deploy the stack will help customers get things up and running.

Reply:
We can also add a quick start link. I am not sure how that works though, as we need to host somewhere on S3. Is there a public bucket available for that?